### PR TITLE
Update samples & docs with SDK version 1.2.0

### PR DESCRIPTION
**Pull request by cai22r**
* Bump SDK version
* Update changelog
* Update iOS sdksample readme
* Update podfile locks
* Update Android Projects. to use AndroidX

### CODE REVIEW

File: iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift [hunk 1]
```diff
@@ -121,7 +128,7 @@ class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSo
     }
 
     func updateMapStyle() {
-        mapView.setStyleSheet(currentStyle.styleSheet)
+        mapView.styleSheet = (currentStyle.styleSheet)
```
Comment (sakthiyuvaraj) [inline iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift:131]: mapView.styleSheet = currentStyle.styleSheet

File: iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift [hunk 2]
```diff
@@ -101,6 +101,13 @@ class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSo
                 self.addPin(atLocation: location!, withTitle: "")
                 return true
             }
+            else
+            {
```
Comment (JuanPapas) [inline iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift:105]: Can you please remove this from the sample?

Comment (sakthiyuvaraj) [inline iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift:105]: Remove

File: Android/samples/search/app/build.gradle [hunk 3]
```diff
@@ -2,11 +2,11 @@ apply plugin: 'com.android.application'
 apply from: 'secrets.gradle'
 
 android {
-    compileSdkVersion 28
+    compileSdkVersion 29
     defaultConfig {
         applicationId "com.microsoft.maps.sample.search"
         minSdkVersion 19
-        targetSdkVersion 28
+        targetSdkVersion 29
```
Comment (JuanPapas) [inline Android/samples/search/app/build.gradle:9]: Bump this to 30, since tests have been verified against Android 11?

**Review by JuanPapas — CHANGES_REQUESTED** [review]

File: iOS/samples/search/BingMapsSearchSample/ViewController.swift [hunk 4]
```diff
@@ -94,7 +94,7 @@ class ViewController: UIViewController, UIPickerViewDelegate {
     }
 
     func updateMapViewColorScheme() {
-        mapView.setStyleSheet(self.traitCollection.userInterfaceStyle == .dark ?
+        mapView.styleSheet = (self.traitCollection.userInterfaceStyle == .dark ?
```
Comment (sakthiyuvaraj) [inline iOS/samples/search/BingMapsSearchSample/ViewController.swift:97]: Remove braches

**Review by sakthiyuvaraj — APPROVED** [review]

**Review by JuanPapas — APPROVED** [review]

**Pull request merged**